docs/developer-docs/ai/inference.mdx

```diff
@@ -0,0 +1,63 @@
+---
+keywords: [intermediate, concept, AI, ai, deAI, deai]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Decentralized AI Inference
+
+<MarkdownChipRow labels={["Intermediate", "Concept", "DeAI" ]} />
+
+It's possible for canister smart contracts to run inference in a number of ways, depending on the decentralization and performance requirements.
+
+## 1. Inference on-chain
+
+Currently, ICP supports on-chain inference of small models using AI libraries such as [Sonos Tract](https://github.com/sonos/tract) that compile to WebAssembly.
+Check out the [image classification example](/docs/current/developer-docs/ai/ai-on-chain) to learn how it works.
+The long-term [vision of DeAI on ICP](https://internetcomputer.org/roadmap#Decentralized%20AI-start) is to support on-chain GPU compute to enable both training and inference of larger models.
```

Comment:
This sentence is repeated from the Overview. It is not necessary to repeat the same exact wording here.